lib/jsdom/living/nodes/HTMLSelectElement-impl.js

```diff
@@ -90,6 +90,12 @@ class HTMLSelectElementImpl extends HTMLElementImpl {
     return this.options.length;
   }
 
+  get selectedOptions() {
+    return createHTMLCollection(this, () => domSymbolTree.treeToArray(this, { filter(node) {
```

Comment:
Oh wow, I totally skipped over the [SameObject]. You're right; we should store it in a private var (and probably create it lazily). And add tests for that!!